steps/bitrise-to-appscan/0.2.0/step.yml

```diff
@@ -0,0 +1,72 @@
+title: AppScan Security Analysis
+summary: Performs security analysis using the IBM AppScan Cloud.
+description: |
+  Performs security analysis using the IBM AppScan Cloud.
+website: https://github.com/ipiranga-atq/bitrise_appscan_step
+source_code_url: https://github.com/ipiranga-atq/bitrise_appscan_step
+support_url: https://github.com/ipiranga-atq/bitrise_appscan_step
+published_at: 2018-09-26T15:58:54.306065087-03:00
+source:
+  git: https://github.com/ipiranga-atq/bitrise_appscan_step.git
+  commit: 96515265be0c324bcd00d1ad96a83b2a6fe2677d
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/ipiranga-atq/bitrise_appscan_step
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: true
+run_if: ""
+inputs:
+- app_path: $BITRISE_APK_PATH
+  opts:
+    description: |
+      Type of application's installer (APK or IPA)
+    is_expand: true
+    is_required: true
+    title: 'Choose file type: APK/IPA'
+    value_options:
```

Comment:
i see your point but it is a bit strange to use `BITRISE_APK_PATH` and `BITRISE_IPA_PATH`.
Value Options means that you need to choose from the listed options, but what if someone uses his own script to build an ipa/apk file and uses different env var name?

I would recommend mentioning that `BITRISE_APK_PATH` or `BITRISE_IPA_PATH` can be used to fill this value, but i would ask you remove the `value_options ` from this input.